src/OpenLoco/src/Interop/Hooks.cpp

```diff
@@ -976,4 +976,15 @@ void OpenLoco::Interop::registerHooks()
             regs = backup;
             return 0;
         });
+
+    registerHook(
+        0x004AFFF3,
+        [](registers& regs) FORCE_ALIGN_ARG_POINTER -> uint8_t {
+            registers backup = regs;
+            auto* entity = reinterpret_cast<OpenLoco::Vehicles::VehicleBogie*>(regs.esi);
```

Comment:
To match the rest of the hooks we would normally write this as `X86Pointer<OpenLoco::Vehicles::VehicleBogie>(regs.esi)`